UefiCpuPkg/Library/MtrrLib/DetectCcNull.c

```diff
@@ -11,21 +11,12 @@
 
 **/
 
-#include <Library/BaseLib.h>
 #include "DetectCc.h"
 
 BOOLEAN
 CcMode (
   VOID
   )
 {
-  STATIC BOOLEAN  mCcMode     = FALSE;
-  STATIC BOOLEAN  mCcModeRead = FALSE;
-
-  if (!mCcModeRead) {
-    mCcMode     = TdIsEnabled ();
-    mCcModeRead = TRUE;
-  }
-
-  return mCcMode;
+  return FALSE;

```

Comment:
Ah, it's just temporary, probably to not break things in the middle of the reorganization?